manifests/dashboard.pp

```diff
@@ -14,6 +14,16 @@
   $dashboard_server_host = '0.0.0.0',
   $dashboard_server_hosts = "https://${indexer_server_ip}:${indexer_server_port}",
 
+  # Parameters used for sso login
+  $enable_sso_login = undef,
```

Comment:
The variable is named "enable_sso_login" but the code manage only openid.
With this parameter name, I expect to be able to use also SAML.